source/simulator/parameters.cc

```diff
@@ -757,20 +757,26 @@ namespace aspect
                          Patterns::Integer (0),
                          "The number of adaptive refinement steps performed after "
                          "initial global refinement but while still within the first "
-                         "time step.");
+                         "time step. These refinement steps (n) are added to the value "
+                         "for initial global refinement (m) so that the total "
+                         "refinement level is n+m.   ");
       prm.declare_entry ("Time steps between mesh refinement", "10",
                          Patterns::Integer (0),
                          "The number of time steps after which the mesh is to be "
                          "adapted again based on computed error indicators. If 0 "
                          "then the mesh will never be changed.");
       prm.declare_entry ("Refinement fraction", "0.3",
                          Patterns::Double(0., 1.),
-                         "The fraction of cells with the largest error that "
-                         "should be flagged for refinement.");
+                         "Cells are sorted from largest to smallest by their total error "
+                         "(determined by the Strategy). Then the cells with the largest "
+                         "error (top of this sorted list) that account for $v$ fraction "
+                         "of the error are refined.");
       prm.declare_entry ("Coarsening fraction", "0.05",
                          Patterns::Double(0., 1.),
-                         "The fraction of cells with the smallest error that "
-                         "should be flagged for coarsening.");
+                         "Cells are sorted from largest to smallest by their total error "
+                         "(determined by the Strategy)  Then the cells with the smallest " 
+                         "error (bottom of this sorted list) that account for v fraction "
```

Comment:
```suggestion
                         "error (bottom of this sorted list) that account for $v$ fraction "
```
Neither here nor above is `v` actually introduced. Maybe just say "the given fraction"?